sites/gxcontractor.com/config/dragon-forms.js

```diff
@@ -4,6 +4,8 @@ const config = new DragonFormsConfig({ url: 'https://endeavor.dragonforms.com' }
 
 config
   .addForm('doNotSell', { omedasite: 'EBM_DoNotSell' })
+  .addForm('newsletterSubscribe', { omedasite: 'GXCnewpref' })
+  .addForm('newsletterManage', { omedasite: 'GXCPrefPage' })
   .addForm('newsletterSignup', { omedasite: 'GXC_eNLshort' });
```

Comment:
Is `newsletterSignup` being used anywhere else?  If not, and if this is going to be the new norm, then we might as well remove it if `newsletterSubscribe` and `newsletterManage` are replacing it.

Reply:
There are three different ones because of the query params added on to some sites. Where they want the query param to show up on the block form, but not on the subscribe page form. 'newsletterSignup' is used in the omeda block form packages/shared/components/blocks/omeda-newsletter-signup.marko, and the others are used on the subscribe page and navigations. Although sometimes brands want one of the query params on a navigation instead.

An example is aviationpros: sites/aviationpros.com/config/dragon-forms.js

Reply:
Sounds good, let's leave `newsletterSignup` here then 👍